api-report/container-runtime.api.md

```diff
@@ -430,7 +430,7 @@ export interface IPendingLocalState {
 
 // @public
 export interface IPendingMessage {
```

Comment:
does this even need to be exported? could we consider deprecating the whole class, and just not exporting it anymore?

Reply:
@wes-carlson Is there a plan to make `IPendingRuntimeState` exported by runtime? Looks like it's currently casting from an `unknown` type

Reply:
No, `IPendingRuntimeState` is internal to runtime and shouldn't be exported.